hasura.planx.uk/migrations/1748350699031_run_sql_migration/up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE VIEW "public"."planx_website_stats" AS
+SELECT
+  (
+    SELECT
+      COUNT(*)
+    FROM
+      (
+        SELECT
+          "public"."analytics_summary"."analytics_id" AS "analytics_id",
+          COUNT(*) AS "count"
+        FROM
+          "public"."analytics_summary"
+        WHERE
+          (
+            "public"."analytics_summary"."analytics_created_at" >= DATE_TRUNC('week', (NOW() + INTERVAL '-1 week'))
+          )
+          AND (
+            "public"."analytics_summary"."analytics_created_at" < DATE_TRUNC('week', NOW())
+          )
+        GROUP BY
+          "public"."analytics_summary"."analytics_id"
+        ORDER BY
+          "public"."analytics_summary"."analytics_id" ASC
+      ) AS "source"
+  ) AS users_last_week,
```

Comment:
Source: https://metabase.editor.planx.uk/question/674-visitors-in-the-last-week/notebook

Originally I had considered querying Metbase directly but that would create a weird invisible dependency (as well as just being more complex).

Note: the "last week" filter in Metabase is querying the previous 7 days, not `now() - 7 days`. This means this is current as of midnight last night.

<img width="637" alt="image" src="https://github.com/user-attachments/assets/a5b2db16-a073-454a-8a6f-1256fe37e4bf" />